rllib/agents/trainer_config.py

```diff
@@ -170,7 +172,8 @@ def __init__(self, trainer_class=None):
         self.input_ = "sampler"
         self.input_config = {}
         self.actions_in_input_normalized = False
-        self.input_evaluation = ["is", "wis"]
+        self.input_evaluation = [ImportanceSamplingEstimator,
```

Comment:
loving this! the more cryptic acronyms we can replace, the better. Having said that, maybe we can settle for a bit of a compromise here in terms of brevity. My suggestion would be:

```python
from ray.rllib.offline import ImportanceSampling
from ray.rllib.offline import WeightedImportanceSampling
```

i.e. import everything in `offline` and drop the `Estimator`. wdyt?

Reply:
changed